lib/cron.js

```diff
@@ -2,7 +2,7 @@
 	if (typeof define === 'function' && define.amd) {
 		define(['luxon'], factory);
 	} else if (typeof exports === 'object') {
-		module.exports = factory(require('luxon'), require('child_process'));
+		module.exports = factory(require('luxon'), (typeof window === 'undefined' ? require('child_process') : {}));
```

Comment:
in browsers, `typeof exports` is `undefined` not `object`, so would the execution even get as far as your logic?

Reply:
I'm not sure but this was required for me. My guess it's because im running vue with vite.